README.md

```diff
@@ -1 +1,29 @@
+
 # workers-py
+
+A set of libraries and tools for Python Workers.
+
+
+## Pywrangler 
+
+A CLI tool for managing vendored packages in a Python Workers project.
+
+### Installation
+
+TODO
```

Comment:
How do I install?
Asking for real, I want to try it out

Reply:
I think you're supposed to be able to install it globally via `sudo uv pip install --system -e .` but my homebrew install breaks that.

So I've just been running it like so:

```
uv run --project $PATH_TO_REPO $PATH_TO_REPO/src/pywrangler/cli.py sync
```